### PR TITLE
PgHero for vets-api on K8s

### PR DESCRIPTION
**Pull request by LindseySaari**
This adds the initial code to run pghero on k8s for vets-api. PgHero currently takes a significant time during bundle install and adds 1-200MB to the vets-api bundle cache. This PR adds the necessary configurations along with instructions on how to run locally via k8s. The guide [here](https://github.com/ankane/pghero/blob/master/guides/Docker.md#deployment) was followed during the initial setup phase

Todo: Customization for Multiple Databases

https://github.com/department-of-veterans-affairs/va.gov-team/issues/32597

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+6. Local port fowarding: `kubectl port-forward name-of-your-pod-here 8080:8080`
+7. Visit `localhost:8080` for the PgHero UI
+
+
+
+
+
+
+
+
```
Comment (rileyanderson) [inline README.md:30]: Don't need extra lines here

File: pghero-pod.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        volumeMounts:
+        - name: pghero-configmap
+          mountPath: /app/config/pghero.yml
+          readOnly: true
+          subPath: pghero.yml
+      volumes:
+      - name: pghero-configmap
+        configMap:
+          defaultMode: 0644
+          name: pghero
```
Comment (rileyanderson) [inline pghero-pod.yml:34]: Add new line char here

**Review by rileyanderson — CHANGES_REQUESTED** [review]
Looks good! Just some formatting changes 😄

File: README.md (hunk 1 above)
Comment (LindseySaari) [reply README.md:30]: oof good find lol!

**Review by rileyanderson — APPROVED** [review]

**Pull request merged**